storage_text_field/fields.py

```diff
@@ -78,18 +80,16 @@ def deconstruct(self):
         return name, path, args, kwargs
 
     def get_file_path(self, value):
-        str_self = str(self).encode('utf-8')
-        digest = hashlib.sha224(
-            str_self + six.text_type(value).encode('utf-8')
```

Comment:
This was sometimes causing problems with boto. So that makes it hard to test without boto keys (which we aren't going to add to a public repo).